README.md

```diff
@@ -116,10 +123,13 @@ optional arguments:
                  Default port is 8001
 ```
 
-On the control machine first edit the [max-init.txt](https://github.com/omarcostahamido/PVM/blob/main/max-init.txt) file. For each Pi device add a numbered line with the video filename, ip, and port. As in:
+On the control machine first edit the [max-init.txt](https://github.com/omarcostahamido/PVM/blob/main/max-init.txt) file. For each Pi device add a numbered line with the video filename, ip, and port. 
+By default, we use port 8001 to receive UDP for our main display. 8002 is used for additional displays. 
+As in:
```

Comment:
It is not only the default port, it is also mandatory to leave it as 8001 for the first display, if the user wants to use 2 displays on the same device.
Also: not 8002 but any other port

Reply:
Done here. https://github.com/omarcostahamido/PVM/commit/a8fb622bbfde7faf534e81d919ba87bc98131cb1